frontend/src/pages/Home.tsx

```diff
@@ -148,6 +180,22 @@ function Home() {
               </section>
             )}
 
+            <section className="my-5">
+              <h2 className="h5 font-weight-normal text-body">Funds Pull Pre Approvals Requests</h2>
+              <ul className="list-group my-4">
+                <FundsPullPreApprovalsList
+                  approvals={approvals}
+                  displayApproveRejectButtons={true}
+                  displayRevokeButton={false}
+                />
+                <li className={classNames(bottomStyles)}>
```

Comment:
Why list group with 1 item?

Reply:
@vladkucherov - I wanted the "See All Approvals" to be identical to transaction's "See All", I didn't want to do it inside FundsPullPreApprovalsList (as implemented in transactions) since it's not part of it to my opinion, it's part of the home page and should appear in any case.
As expected, I have no idea if I can have the same result with a different element\tag